src/containers/RecentTrades/Yours.tsx

```diff
@@ -113,30 +47,48 @@ class RecentTradesYoursContainer extends React.Component<Props> {
         ];
     };
 
-    private handleOnSelect = (index: string) => {
-        const { list, currentPrice } = this.props;
+    const retrieveData = () => {
+        return [...list].length > 0
+            ? [...list].map(renderRow)
+            : [[[''], formatMessage({ id: 'page.noDataToShow' })]];
```

Comment:
1. There are 3 columns.
```suggestion
            : [[[''], formatMessage({ id: 'page.noDataToShow' }), ['']]];
```
2. I think better to rework this and pass only one elements like:
```formatMessage({ id: 'page.noDataToShow' });```